driver-core/src/main/com/mongodb/operation/CommitTransactionOperation.java

```diff
@@ -64,7 +66,20 @@ public CommitTransactionOperation(final WriteConcern writeConcern) {
      * @since 3.11
      */
     public CommitTransactionOperation(final WriteConcern writeConcern, final boolean alreadyCommitted) {
+        this(writeConcern, null, alreadyCommitted);
+    }
+
+    /**
+     * Construct an instance.
+     *
+     * @param writeConcern the write concern
+     * @param recoveryToken the recovery token
+     * @param alreadyCommitted if the transaction has already been committed.
+     * @since 3.11
+     */
+    public CommitTransactionOperation(final WriteConcern writeConcern, final BsonDocument recoveryToken, final boolean alreadyCommitted) {
```

Comment:
Since recoveryToken is optional (not always required) it can be set via a setter rather than an additional constructor.

Reply:
Done